Math/LCM.txt

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    int n1, n2, max;
+    cout << "Enter two numbers: ";
+    cin >> n1 >> n2;
```

Comment:
Kindly avoid taking the user input. Define your variables in the code. This makes the code easy to understand.